vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -631,6 +644,25 @@ class RoomDetailFragment @Inject constructor(
         }
     }
 
+    /**
+     * Set the visual state of the call buttons (voice/video) to enabled/disabled according to user permissions
+     *
+     * @param canManageWidgets true if the user is allowed to manage widgets (relevant for group calls)
+     * @param allowedToStartWebRTCCall true if the user is allowed to start a WebRTC call in a room with two participants
+     * @param asyncRoomSummary the room summary
+     */
+    private fun setCallButtonsState(canManageWidgets: Boolean, allowedToStartWebRTCCall: Boolean, asyncRoomSummary: Async<RoomSummary>) {
+        val callButtonsEnabled = when (asyncRoomSummary.invoke()?.joinedMembersCount) {
+            1 -> false
+            2 -> allowedToStartWebRTCCall
+            else -> canManageWidgets
+        }
+        val voiceButton = optionsMenu?.findItem(R.id.voice_call)
+        val videoButton = optionsMenu?.findItem(R.id.video_call)
+        voiceButton?.icon?.alpha = if (callButtonsEnabled) 255 else 60
+        videoButton?.icon?.alpha = if (callButtonsEnabled) 255 else 60
```

Comment:
Small remark, maybe use hexa values (`0xFF`, ...), but it's really a minor remark!